scalableminds/webknossos
# Storing user bounding box with tracing

### Mailable description of changes:
 - The user-specified bounding box is now saved with the tracing
 - It is also exported to and parsed from NML files
 - When merging multiple tracings (e.g. uploading multiple NML files), the resulting tracing always has *no* user bounding box

### Steps to test:
- Create explorational tracing
- In Settings → Other, specify a bounding box
- Close and reload the tracing, bounding box should still be there

### Note
- involves [new braingames-libs version](https://github.com/scalableminds/braingames-libs/tree/userBoundingBox), compare [wiki page](https://github.com/scalableminds/webknossos-wiki/wiki/braingaims-libs-versioning)

### Issues:
- fixes #2043

------
